app/controllers/api/v1/posts_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V1::PostsController < ApplicationController
+  before_action :find_user
+
```

Comment:
:red_circle: [REQUIRED] In the context of your API controllers, it's essential to apply the following filters consistently across all the api's controllers

```
 skip_before_action :verify_authenticity_token, only: [:create]
  skip_before_action :authenticate_user!, except: []
```

It's important to note that APIs require a different approach to authentication compared to traditional web applications. API clients don't maintain sessions like web browsers, so you typically rely on tokens or other stateless mechanisms. Since JWT authentication is not yet implemented, applying these filters allows you to build and test your API routes while ensuring they work as expected.

However, please keep in mind that security is a top priority when working with APIs. Once you implement JWT authentication or any other custom authentication method you can remove these filter.

Reply:
Thank you for the information Sir

Reply:
This change has been done well